modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -97,7 +134,7 @@ def apply(self, func, *args, **kwargs):
             self._is_debug(log) and log.debug(
                 f"SUBMIT::_apply_list_of_funcs::{self._identity}"
             )
-            result, length, width, ip = _apply_list_of_funcs.remote(call_queue, data)
+            result, length, width, ip = self._apply_call_queue(call_queue, data)
```

Comment:
Why not?
```suggestion
            result, length, width, ip = _apply_list_of_funcs.remote(data, *deconstruct_call_queue(call_queue))
```

Reply:
well, this way the futures are being passed incorrectly:
```python
def func(a, b, *futures):
    print(a)
    print(b)
    print(futures)


def return_sequence():
    return 1, 2, [7, 8, 9]

func(*return_sequence())
# 1
# 2
# ([7, 8, 9],) <-- we don't want a list to be nested into a tuple here
```

Reply:
We could unpack inside `return_sequence`:
```python
def return_sequence():
    return 1, 2, *[7, 8, 9]
```

Reply:
done